api/disabled_parameters.go

```diff
@@ -0,0 +1,151 @@
+package api
+
+import (
+	"fmt"
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/labstack/echo/v4"
+)
+
+// DisabledParameter holds the name and disabled status of a parameter
+type DisabledParameter struct {
+	Name     string
+	Disabled bool
+}
```

Comment:
This could be `map[string]bool` instead of a struct. With that format, you could iterate over the set parameters and check if they're enabled or not.

The current algorithm iterates over the disabled parameters and checks if they were used. In the worst case, something like a transaction search could have something like 20 checks even if the user didn't set any parameters.

Reply:
Done